PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+# Tinted Theming Project Governance
+
+Welcome to the Tinted Theming project! This document outlines the
+governance structure and procedures for maintaining and contributing to
+this theming driven community.
+
+## 1. Project Governance and Final Decision Authority
+
+The Tinted Theming project is currently governed by three admins:
+
+- [@balek], who has been actively involved in the community since the
+  original base16 theming project.
+- [@JamyGolden], who has been actively involved in the formation of
+  base16-project and tinted-theming.
+- [@FredHappyface], who recently joined the administration team.
+
+**Final Say:** The three admins collectively have the final say on all
+matters related to the overall direction, policies, and major decisions
+concerning the project. This ensures that the project remains true to
```

Comment:
```suggestion
**Final Say:** The admins collectively have the final say on all
matters related to the overall direction, policies, and major decisions
concerning the project. This ensures that the project remains true to
```

Reply:
We may or may not add more admins in the future, so I'd like to keep this generic